.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: node_js
 
 node_js:
-  - '0.10'
```

Comment:
I want to keep on testing node 0.10 for now. 
There are certain restrictive packaging environments where you're stuck with old versions.